evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -70,6 +71,7 @@ contract Operator is
     Owned(owner)
   {
     linkToken = LinkTokenInterface(link); // external but already deployed and unalterable
+    new OperatorProxy(link);
```

Comment:
Is there any need for the `Operator` to store a reference to `OperatorProxy` @se3000 ?